mint.json

```diff
@@ -95,16 +92,20 @@
       ]
     },
     {
-      "group": "Advanced Features",
-      "pages": ["projects", "variables"]
+      "group": "Account & Views",
```

Comment:
weird grouping these together tbh 😄

Reply:
How would you call it? it makes no sense to make a category for 2 sites ... or just one. Even changelog is not ok, to hang there alone. `Best practices` will grow soon with use case specific best practices. hopefully.